packages/protocol/lib/registry-utils.ts

```diff
@@ -42,7 +44,6 @@ export enum CeloContractName {
 }
 
 export const usesRegistry = [
-  CeloContractName.Escrow,
```

Comment:
just checking this?  @martinvol

Reply:
Yes this is intended, details:

- in this release (8), `Escrow` moved from using the `UsingRegistry` mixin -> `UsingRegistryV2BackwardsCompatible`. The main difference in these is that `UsingRegistryV2BackwardsCompatible` has the registry address stored as a constant (called `registryContract`) into the mixin instead of using a storage variable to store it.
- `usesRegistry` in `registry-utils` is only used in `assertRegistryAddressesSet` (in `packages/protocol/lib/test-utils.ts`), and it just checks that the registry address was set properly for the contract in the `registry` variable; this will not always succeed because `registry` is just a dummy variable in `UsingRegistryV2BackwardsCompatible` in order to maintain storage alignment 😬 (Unfortunately, we couldn't rename the vars without breaking release tooling).

Let me know if that makes sense or if something seems off here! 😄